### ISSUE TITLE
facebook-limited-login backend is failing to verify the JWT signature

### ISSUE THREAD

**Issue by jkeyes** [python-social-auth/social-core#904]
### Expected behaviour

The facebook-limited-login backend is failing to verify the JWT signature.

In my code I call:

```Python
user = request.backend.do_auth(auth_token)
```

where `auth_token` is the JWT access token retrieved by the Facebook iOS SDK. The following is an example of the payload of that token:

```
{
  "iss": "https://www.facebook.com",
  "aud": "{aud}",
  "sub": "{sub}",
  "iat": {iat},
  "exp": {exp},
  "jti": "{jti}",
  "nonce": "{nonce}",
  "email": "{email}",
  "given_name": "{given_name}",
  "family_name": "{family_name}",
  "name": "{name}",
  "picture": "{picture_url}"
}
```

The kid from the JWT is in the JWKS JSON from https://www.facebook.com/.well-known/oauth/openid/jwks/.

### Actual behaviour

JWT verification fails.

### What are the steps to reproduce this issue?

I don't have an easy way to reproduce given actual user details would need to be revealed.

### Any logs, error output, etc?

```
  File ".../site-packages/social_core/backends/facebook_limited.py", line 29, in authenticate
    kwargs["response"] = self.validate_and_return_id_token(raw_jwt, "")
  File ".../site-packages/social_core/backends/open_id_connect.py", line 220, in validate_and_return_id_token
    raise AuthTokenError(self, "Signature verification failed")
social_core.exceptions.AuthTokenError: Token error: Signature verification failed
```

### Any other comments?

I can see social-core is retrieving the keys from Facebook's JWKS so I am wondering am I missing something obvious here or is there a bug? I've checked the docs but there isn't any Limited Login specific docs on the site.

**Comment by jkeyes** [comment]
Of course since I posted this I got it to work. I'm not clear on what caused the verification issue above. I tested the verification by doing the following:

```Python
import jwt
from jwt import PyJWKClient
token = "{token}"
kid = "{kid}"
url = "https://www.facebook.com/.well-known/oauth/openid/jwks/"
jwks_client = PyJWKClient(url, headers=optional_custom_headers)
signing_key = jwks_client.get_signing_key_from_jwt(token)
data = jwt.decode(
    token,
    signing_key.key,
    algorithms=["RS256"],
    audience="{aud}",
)
print(data)
```
and that worked as expected.

I then did some debugging and noticed I had not set `SOCIAL_AUTH_FACEBOOK_LIMITED_LOGIN_KEY` in `settings.py`. I already had the `SOCIAL_AUTH_FACEBOOK_KEY` setting in place, so my brain overlooked the need to set it again for limited login support.

With this change in place everything worked as expected.

I _think_ this is the only setting needed, I will hopefully get around to contributing a documentation PR for the limited login backend later today.
